client/app/bundles/shared/components/Blockquote.css

```diff
@@ -0,0 +1,15 @@
+.App {
+  text-align: center;
+  background-color: rgba(0, 0, 0, 0.25);
+  font-family: Hiragino Sans, Lato ;
```

Comment:
Let's make Lato come first here and not use Hiragino Sans! Could probably be `Lato, sans-serif`.